integration/tests/basic/test_EIP3448.py

```diff
@@ -0,0 +1,22 @@
+import allure
+from integration.tests.basic.helpers.basic import BaseMixin
+
+
+@allure.feature("EIP Verifications")
+@allure.story("EIP-3448: MetaProxy Standard: \
+              A minimal bytecode implementation for creating proxy contracts \
+              with immutable metadata attached to the bytecode")
+class TestMetaProxyStandard(BaseMixin):
+    def test_create_from_call_data(self, meta_proxy_contract):
```

Comment:
Please add more assertion checks. Because in case like: "CalldataReturnMultiValues" or "ReturnRevert" it's very important to check values

Reply:
Done. Please, check if it is ok